rackcity/models/it_instance.py

```diff
@@ -1,10 +1,23 @@
+import re
+from django.core.exceptions import ValidationError
 from django.db import models
 from .it_model import ITModel
 from .rack import Rack
 
 
+def validate_hostname(value):
+    hostname_pattern = re.compile("[a-z][a-z0-9-]{0,61}[a-z0-9]")
```

Comment:
thank you, patrick wang

Reply:
Did you get this regex from somewhere or write it based on the rule?
1. This will require hostname to be at least two characters long -- is `a` not valid?
2. Can we document either on notion or in the comments the plain english version of what we think is a valid hostname

Reply:
fixing regex, will add to documentation